### DISCUSSION TITLE
Howto navigate to different Tab stack?

### DISCUSSION THREAD

**Discussion by rtpm** [expo/router#548] [Q&A]
Hello,

My app (expo-router v. 1.5.1) is built on top of 2 Tab Stacks. In the main app directory there' s notificationScreen.tsx which renders user notifications from the icon at the top of the header bar. There are 2 tabs:

```
app
    /(auth) 
    /(tabs)
         /products/[productid]/view
         /orders
    /notificationScreen.tsx (presentation="modal")
```

In the notification there's a button so when clicked it should redirect to order details which points to:
/products/productid/details/ 

router = useRouter() from expo-router
router.push('/products/30456/details')
(the same for replace)

It works fine if I open notifications and press the notification button while being in the "products" stack.
When I do while "being" in "orders" stack, I am redirected to the proper screen, but no "Back" header button is available.
It doesn't matter which screen should the "Back button" point to. if it just didn't block the "products" stack as I can not move anywhere else there.

Can this be done somehow?
Thanks

**Comment by gerynugrh** [comment]
Hi @rtpm did you figure out a way to solve these issues? I'm facing a similar problem currently

**Comment by rotemitz** [comment]
Just came across this, not sure if you ever got to solve this, but for the chance you didn't, you can just put the back button manually on the stack screen like this:
```
<Stack.Screen
          name="{screen-name}"
          options={{
            headerShown: true,
            headerLeft: () => (
              <Icon
                name="arrow-left-s-line"
                size={24}
                onPress={() => {
                  router.dismiss();
                }}
              />
            )
          }}
        />
